scripts/test-nanvixd.sh

```diff
@@ -1,50 +1,30 @@
 #!/bin/bash
 
+set -euo pipefail
+
 NANVIXD_SOCKADDR=$1
-LINUXD_SOCKADDR=$2
-SANDBOX_SOCKADDR=$3
-PROGRAM_NAME=$4
-PROGRAM_ARGS=$5
-PROGRAM_EXPECTED_OUTPUT=$6
-TIMEOUT=${7:-90}
+PROGRAM_NAME=$2
+PROGRAM_ARGS=$3
+PROGRAM_INPUT=$4
+PROGRAM_EXPECTED_OUTPUT=$5
+TIMEOUT=${6:-90}
 
 NANVIX_HOME=`git rev-parse --show-toplevel`
 LOGS_DIR=${NANVIX_HOME}/logs/nanvixd-$(basename "${PROGRAM_NAME}")
 
-mkdir -p ${LOGS_DIR}
-
-kill_children() {
-    local parent_pid=$1
-
-    if [ -z "$parent_pid" ]; then
-           return
-    fi
-
-    # Find all child processes
-    local children=$(pgrep -P $parent_pid)
-    for child in $children; do
-        # Recursively call the function for each child process
-        kill_children $child
-    done
+# Parameters for the requests to nanvixd.
+TENANT_ID="foo"
+APP_NAME="bar"
 
-    # Kill the parent process
-    sudo /usr/bin/pkill -e -INT -P $parent_pid
-}
+mkdir -p ${LOGS_DIR}
 
 # Run nanvixd.
-NANVIXD_STDOUT_FILE_NAME="${LOGS_DIR}/nanvixd-stdout_$(date "+%Y_%m_%d_%H_%M").log"
-NANVIXD_STDERR_FILE_NAME="${LOGS_DIR}/nanvixd-stderr_$(date "+%Y_%m_%d_%H_%M").log"
 CONSOLE_FILE_NAME="${LOGS_DIR}/kernel_$(date "+%Y_%m_%d_%H_%M").log"
-RUST_LOG=trace timeout -s SIGINT --preserve-status --foreground ${TIMEOUT} \
+timeout -s SIGINT --preserve-status --foreground ${TIMEOUT} \
```

Comment:
We are no longer running with trace enabled, which impacts debuggability in the test pipeline. Is this intentional?

Reply:
It was not intentional. Thanks for pointing it out.